<launch>
    <node type="rrt_planner" name="rrt_planner" pkg="planner" output="screen">
        <param name="map_topic" value="/map"/>
        <param name="goal_topic" value="/goal"/>
        <param name="pose_topic" value="/pose"/>
        <param name="max_vertices" value="1500"/>
        <param name="step_size" value="20"/>
    </node>
</launch>
